package.json

```diff
@@ -26,11 +26,14 @@
     "@uniswap/v2-core": "^1.0.1",
     "chai": "^4.2.0",
     "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-no-only-tests": "^2.4.0",
+    "eslint-plugin-prettier": "^3.1.4",
     "ethereum-waffle": "^3.1.0",
     "lodash": "^4.17.20",
     "mocha": "^8.1.3",
-    "prettier": "^2.1.1",
+    "prettier": "^2.1.2",
```

Comment:
Are `.prettierignore` and `.prettierrc.js` intentionally left empty?

Reply:
not intentionally.. I meant to use them, but I guess we don't need them.